### PR TITLE
Add NaNs to estimation result and handle NaNs in graphs

### PR DESCRIPTION
**Pull request by karlb**
All graphs will now display an edge if it is not NaN, even if the values is zero. This makes the model relations visible even in cases where the actual effect is zero.

For this to work, adding NaNs to the estimation results was necessary, as the graphs would explode in size, otherwise.

### CODE REVIEW

**Comment by karlb** [comment]
~~@Holger-Bartel I think we have set the final value to NaN. Is this a logical consequence and we can get the value from a different place, or did we introduce an error, here?~~

**Comment by karlb** [comment]
Ignore that, I am just skipping that part in the graph rendering because that node has no outgoing edges.

EDIT: fixed now.

File: causing/graph.py [hunk 1]
```diff
@@ -115,10 +92,12 @@ def dot(
             dot_str += '         "{}" -> "{}" [label = "{}"{}];\n'.format(
                 xnode, ynode, wei_str, col_str
             )
+            xnodes_with_edge.add(xnode)
```
Comment (Holger-Bartel) [inline causing/graph.py:95]: I don't get this. For me it seems that *all* nodes are added xnodes_with_edge?

File: output/education/estimate.json [hunk 2]
```diff
@@ -143,52 +143,52 @@
...
+            NaN
         ]
     ],
     "eyy_hat_std": [
         [
-            0.0,
-            0.0,
+            NaN,
+            NaN,
             NaN
```
Comment (Holger-Bartel) [inline output/education/estimate.json:187]: What does a black Nan in the diff means?

**Review by Holger-Bartel — CHANGES_REQUESTED** [review]

File: causing/graph.py (hunk 1 above)
Comment (Holger-Bartel) [reply causing/graph.py:95]: Ok, I didn't see the continue..

**Review by Holger-Bartel — APPROVED** [review]

File: output/education/estimate.json (hunk 2 above)
Comment (Holger-Bartel) [reply output/education/estimate.json:187]: OK it's not part of the diff, just a non-valid json entry

**Pull request merged**